apps/nextjs-website/src/app/page.tsx

```diff
@@ -6,6 +6,25 @@ import Layout from '@/components/organisms/Layout/Layout';
 import News from '@/components/organisms/News/News';
 import ProductsShowcase from '@/components/organisms/ProductsShowcase/ProductsShowcase';
 import { getProducts } from '@/lib/api';
+import { Metadata } from 'next';
+
+export async function generateMetadata(): Promise<Metadata> {
+  const { homepage, shared } = translations;
+
+  return {
+    title: shared.siteTitle,
+    description: homepage.productsShowcaseTitle,
+    openGraph: {
+      title: shared.siteTitle,
+      description: homepage.productsShowcaseTitle,
+      url: '/',
+    },
+    twitter: {
+      card: 'summary_large_image',
+      site: shared.siteTitle,
```

Comment:
According to [Twitter's documentation about meta tags](https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started), the value of the `twitter.site` property should be "@username for the website used in the card footer."